Tools/Scripts/generate-static-analysis-archive

```diff
@@ -63,8 +63,10 @@ def generate_results_page(project_dirs, id_string, output_root, static_analysis_
                 project_issue_count=get_project_issue_count_as_string(static_analyzer_index_path),
                 project_name=project_name)
 
-    if is_new_result:
+    if result_type == 'new':
         heading = 'New issues in {}'.format(id_string)
+    elif result_type == 'unexpected':
+        heading = 'Unexpected issues from {}'.format(id_string)
```

Comment:
Isn't this just `'{} issues from {}'.format(result_type, id_string)` in general?

Reply:
Yes, thanks! I also applied the same logic change to `create_results_file`.